wikimedia/wikipedia-ios
# Fix On This Day widget link

**Fixes Phabricator ticket:** N/A

### Notes
When tapping an On This Day widget, the On This Day View Controller is supposed to open with the highlighted event int the middle of the screen. This morning I noticed it was broken. Looks like it was a regression that snuck in.

### Test Steps
Add an "On This Day" widget. Note the event being shown. Tap the widget. Ensure the View Controller is scrolled to the event shown on the widget.
